README.md

```diff
@@ -36,4 +36,12 @@ You can get `gonum/plot` using go get:
 
 `go get gonum.org/v1/plot/...`
 
+
+Note: If you get this error during install "Package glesv2 was not found in the pkg-config search path.", install the `libgles-dev` package for your system:
+
+`sudo apt install libgles-dev`
```

Comment:
perhaps redirect (also? instead?) to Gio's official page for installation?
we should probably rely on their instructions than ours as the latter might sync' off...

ie: https://gioui.org/doc/install